pkgs/development/compilers/flutter/update.py

```diff
@@ -0,0 +1,386 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i python3 -p python3Packages.pyyaml
+
+import shutil
+import json
+import urllib.request
+import tempfile
+from sys import exit
+import os
+import subprocess
+import re
+import json
+import argparse
+import yaml
+import json
+
+
+NIXPKGS_ROOT = subprocess.Popen(['git', 'rev-parse', '--show-toplevel'],
+                                stdout=subprocess.PIPE, text=True).communicate()[0].strip()
+
+
+# Return out paths
+def nix_build(code):
+    temp = tempfile.NamedTemporaryFile(mode='w')
+    temp.write(code)
+    temp.flush()
+    os.fsync(temp.fileno())
+
+    process = subprocess.Popen(
+        ["nix", "build", "--impure", "--print-out-paths", "--no-link", "--expr",
```

Comment:
this depends on experimental features. please add the necesarry flags for enabling the features in this command.